tst/testbugfix/2013-03-12-t00285.tst

```diff
@@ -1,5 +1,58 @@
 # 2013/03/12 (MH)
-gap> v:=IdentityMat(28,GF(2))[1];
-<a GF2 vector of length 28>
-gap> v{[1..Length(v)]}{[1..5]};
-Error, List Elements: <lists> must be a list (not a data object)
+#
+# The following tests used to crash. This is because for for compressed FFE
```

Comment:
If you happen to make further changes to this PR, then perhaps change this too. Otherwise, no problem.
```suggestion
# The following tests used to crash. This is because for compressed FFE
```